engine/class_modules/sc_shaman.cpp

```diff
@@ -5272,6 +5280,58 @@ struct thundercharge_t : public shaman_spell_t
   }
 };
 
+// ==========================================================================
+// Fae Transfusion - Night Fae Covenant
+// ==========================================================================
+
+struct fae_transfusion_damage_t : public shaman_spell_t
+{
+  fae_transfusion_damage_t( shaman_t* player, const std::string& options_str )
+    : shaman_spell_t( "fae_transfusion_", player, player->find_spell( 328928 ), options_str )
+  {
+    aoe        = 4;
+    ground_aoe = background = true;
+    dynamic_tick_action     = true;
+    base_tick_time          = 500_ms;
+    dot_duration            = 3_s;
```

Comment:
This dot/tick stuff is unnecessary. The spell doesn't have any ticking damage in its data so it wouldn't do ticking damage anyways.